docker-registry-sync/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# based on https://github.com/xelalexv/dregsy/blob/85b34eb8a0c7ffc1b7c610bc8de1388230a12551/Dockerfile
+FROM xelalex/dregsy@sha256:9b0fe252cb8bf88f67871efd6559ef89b76013c46dfbc49b961d396a4b8d1887
+
+LABEL maintainer "neagu@itis.swiss"
+
+RUN apk add python3
+RUN pip3 install --upgrade pip setuptools
```

Comment:
MINOR: we also tend to upgrade ``wheels`` together with these two. 
MINOR: list every item in a different line (as above)